docs/docs/dev_docs/contracts/syntax/historical_access/history_lib_reference.md

```diff
@@ -0,0 +1,119 @@
+---
+title: History Reference
+---
+
+<!-- Note: This will soon be moved into an Aztec.nr reference category under Aztec.nr smart contracts -->
+
+## Note inclusion 
+
+Note inclusion proves that someone owned a note at a specific block number.
+
+## prove_note_inclusion
+
+`prove_note_inclusion` takes 4 parameters:
+
+| Name            | Type                   | Description                                         |
+|-----------------|------------------------|-----------------------------------------------------|
+| note_interface  | NoteInterface<Note, N> | Interface for the note with necessary functionality|
+| note_with_header| Note                   | The note you are proving inclusion for             |
+| block_number    | u32                    | Block number for proving note's existence           |
+| context         | PrivateContext         | Private context     |
+
+## prove_note_commitment_inclusion
```

Comment:
@benesjan are we still using "commitment" in some places?

Reply:
I can change to note hash which we've done in a few places, but since the library calls it `commitment` I've explained what commitment means here

Reply:
Yea I agree in keeping it similar to the code. Hence question to Jan on why we are using that term.

How do we get the commitment btw? Is it the same as note hash? A user is definitely going to be confused.

Reply:
Yea I agree in keeping it similar to the code. Hence question to Jan on why we are using that term.

How do we get the commitment btw? Is it the same as note hash? A user is definitely going to be confused.

Reply:
afaik it's the same thing - maybe should put a note here?

Reply:
added this - A **commitment**, also referred to as a **note hash** is a public acknowledgment of the existence of a note without revealing the content of the note. You can learn more about how to compress a note to a note hash [here](../../../../concepts/advanced/data_structures/trees.md#example-note).
wdyt?